### PR TITLE
Add ssh-keyscan for authoring and delivery tomcat

### PR DESCRIPTION
**Pull request by phuongnq**
https://github.com/craftercms/craftercms/issues/5708

### CODE REVIEW

File: images/authoring/tomcat/docker-entrypoint.sh [hunk 1]
```diff
@@ -23,6 +23,18 @@ chown_dir() {
   fi
 }
 
+host_keyscan() {
+    if [ ! -d $CRAFTER_SSH_CONFIG ]; then
+        mkdir -p $CRAFTER_SSH_CONFIG
+        chown_dir "$CRAFTER_SSH_CONFIG"
+    fi
+    declare -a DOMAINS=("bitbucket.com" "gitlab.com" "github.com")
```
Comment (avasquez614) [inline images/authoring/tomcat/docker-entrypoint.sh]: Can you maybe make this a env variable that can be overwritten?

File: images/delivery/tomcat/docker-entrypoint.sh [hunk 2]
```diff
@@ -23,6 +23,18 @@ chown_dir() {
   fi
 }
 
+host_keyscan() {
+    if [ ! -d $CRAFTER_SSH_CONFIG ]; then
+        mkdir -p $CRAFTER_SSH_CONFIG
+        chown_dir "$CRAFTER_SSH_CONFIG"
+    fi
+    declare -a DOMAINS=("bitbucket.com" "gitlab.com" "github.com")
```
Comment (avasquez614) [inline images/delivery/tomcat/docker-entrypoint.sh]: Can you maybe make this a env variable that can be overwritten?

**Review by avasquez614 — CHANGES_REQUESTED** [review]
Please also add these changes to the deployer `docker-entrypoint.sh`

File: images/authoring/tomcat/docker-entrypoint.sh (hunk 1 above)
Comment (phuongnq) [reply images/authoring/tomcat/docker-entrypoint.sh]: Updated to have env variable `SSH_KEYSCAN_DOMAINS` with a default value if not set and separated by `,`.

Example to set:

```
export SSH_KEYSCAN_DOMAINS=bitbucket.com,gitlab.com,github.com
```

File: images/delivery/tomcat/docker-entrypoint.sh (hunk 2 above)
Comment (phuongnq) [reply images/delivery/tomcat/docker-entrypoint.sh]: Updated to have env variable `SSH_KEYSCAN_DOMAINS` with a default value if not set and separated by `,`.

Example to set:

```
export SSH_KEYSCAN_DOMAINS=bitbucket.com,gitlab.com,github.com
```

**Comment by phuongnq** [comment]
@avasquez614 Added for deployer.

**Review by avasquez614 — APPROVED** [review]

**Pull request merged**